templates/api.mustache

```diff
@@ -1,4 +1,3 @@
-{{>partial_header}}
```

Comment:
Why is the partial header removed? Does the api-small class extend the api.mustache class or smth?

Reply:
not to include the header in all the classes as it's generated for each subclass, small one is generated once, thus the header stays there

Reply:
ahh okay, so only the init files in the big services have that header?